src/common/libkvs/kvs_checkpoint.c

```diff
@@ -37,7 +37,7 @@ flux_future_t *kvs_checkpoint_commit (flux_t *h,
         timestamp = flux_reactor_now (flux_get_reactor (h));
 
     if (!(f = flux_rpc_pack (h,
-                             "kvs-checkpoint.put",
+                             "content-backing.checkpoint-put",
```

Comment:
Ah good cleanup.  This was initially created as a separate service since it seemed like it ought to be independent of the "content" service, but that's not how it has played out so this change make sense and in fact has been bugging me.